libnmstate/nm/checkpoint.py

```diff
@@ -26,50 +26,28 @@
 from libnmstate.nm import common
 
 
-class NMCheckPointError(Exception):
-    pass
-
-
-class NMCheckPointCreationError(NMCheckPointError):
-    pass
-
-
-class NMCheckPointPermissionError(NMCheckPointError):
-    pass
-
-
```

Comment:
I agree with this, but we must have on mind that this is a breaking change.

Reply:
They are internal errors which will never raise to user.